scripts/ci/Dockerfile.functional

```diff
@@ -1,12 +1,9 @@
-FROM anchore-engine:dev
-
-RUN pip3 install --user tox
+ARG TEST_IMAGE_NAME
+FROM ${TEST_IMAGE_NAME}
 
 USER root
-RUN set -ex && \
-    echo "installing OS dependencies" && \
-    yum update -y && \
-    yum install -y gcc make git python3-wheel python36-devel
+
+RUN yum install -y make gcc python38 python38-devel python36 python36-devel
```

Comment:
Not sure why this is no longer installing `git` and why it removed the title and `-ex`

Reply:
I'll put it back (was it needed to begin with? it doesn't seem like it was)

Reply:
Well I don't know either, so I was hoping you would enlighten the removal :)